code/ensemble/utils/use-ensemble-criteria.R

```diff
@@ -57,7 +58,7 @@ use_ensemble_criteria <- function(forecasts,
   
   # 5. Drop "other" designated models
   if (exclude_designated_other) {
-    not_other <- get_model_designations(here()) %>%
+    not_other <- suppressWarnings(get_model_designations(here())) %>%
```

Comment:
Any particular reason we want to suppress warnings? Just thinking that we might sometimes get something informative about problems with the yaml files that could get lost here.

Reply:
Fair, I was just seeing lots of line ending warnings which didn't have any impact, but I'll remove.